ComfortZoneApp/app/src/main/java/com/example/comfortzone/notification/NotificationHelper.java

```diff
@@ -0,0 +1,53 @@
+package com.example.comfortzone.notification;
+
+import static com.example.comfortzone.ui.HostActivity.REQUEST_CODE;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+
+import androidx.core.app.NotificationCompat;
+
+import com.example.comfortzone.R;
+import com.example.comfortzone.ui.HostActivity;
+
+public class NotificationHelper {
+
+    private Context context;
+    private static final String NOTIFICATION_CHANNEL_ID = "10001";
+    public static final String TITLE = "ComfortZone";
+    public static final String CONTENT = "How's the weather right now?";
+
+    public NotificationHelper(Context context) {
+        this.context = context;
+    }
+
+    public void createNotification() {
```

Comment:
I feel like this can just be a static method - we should avoid storing the context as much as possible, as it may cause memory leak as it can't be properly garbage collected if other class has reference to it